sagemaker-featurestore/feature_store_securely_store_images.ipynb

```diff
@@ -0,0 +1,690 @@
+    "df = cast_object_to_string(df)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create your Feature Group and Ingest your data into it\n",
+    "\n",
+    "Below we start by appending the `EventTime` feature to your data to timestamp entries, then we load the feature definition, and instantiate the Feature Group object. Then lastly we ingest the data into your feature store."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_group_name = 'mnist-feature-group-' + strftime('%d-%H-%M-%S', gmtime())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Instantiate a `FeatureGroup` object for your data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_group = FeatureGroup(name=feature_group_name, sagemaker_session=sagemaker_session)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "record_identifier_feature_name = \"id\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Append the `EventTime` feature to your data frame. This parameter is required, and time stamps each data point."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "current_time_sec = int(round(time.time()))\n",
+    "event_time_feature_name = \"EventTime\"\n",
+    "# append EventTime feature\n",
+    "df[event_time_feature_name] = pd.Series([current_time_sec]*len(df), dtype=\"float64\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load Feature Definition's of your data into your feature group."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_group.load_feature_definitions(data_frame=df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create your feature group.\n",
+    "\n",
+    "**Important**: You will need to substitute your KMS Key ARN for `kms_key` for server side encryption. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=False,\n",
+    "    offline_store_kms_key_id = kms_key # Substitute kms_key with your kms key. \n",
+    ")"
+   ]
+  },
```

Comment:
The notebook indicates that it will demo KMS integration for online or offline. Although the online store is not enabled, I think that we should still include a line in the markdown cell that indicates how to specify a KMS key for SSE, if the online store is enabled. I.e. https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_OnlineStoreSecurityConfig.html#sagemaker-Type-OnlineStoreSecurityConfig-KmsKeyId.

Reply:
Made this change.